contrib/raftexample/kvstore.go

```diff
@@ -40,9 +40,9 @@ type kv struct {
 func newKVStore(snapshotter *snap.Snapshotter, proposeC chan<- string, commitC <-chan *string, errorC <-chan error) *kvstore {
 	s := &kvstore{proposeC: proposeC, kvStore: make(map[string]string), snapshotter: snapshotter}
 	// replay log into key-value map
-	s.readCommits(commitC, errorC)
+	s.readCommits(commitC, errorC, true)
```

Comment:
I think that's why we have another `readCommits` in the goroutine, to handle following snapshots?